.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -0,0 +1,110 @@
+jobs:
+# Configure, build, install, and test job
+- job: 'linux_build'
+  displayName: 'Linux Builds'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  timeoutInMinutes: 120
+  variables:
+    CTEST_OUTPUT_ON_FAILURE: 1
+  strategy:
+    maxParallel: 8
+    matrix:
+      gcc_build:
+        CXX_COMPILER: 'g++'
+        PYTHON_VER: '3.7'
+        C_COMPILER: 'gcc'
+        F_COMPILER: 'gfortran'
+        BUILD_TYPE: 'release'
+
+  steps:
+  - bash: |
+      uname -a
+      free -m
+      df -h
+      ulimit -a
+    displayName: "Hardware Information"
+
+  - bash: |
+      sudo apt-get update
+      sudo apt-get install gfortran
+    displayName: "Install gFortran"
+
+  - bash: |
+        echo "##vso[task.prependpath]$CONDA/bin"
+        conda config --set always_yes yes
+    displayName: "Add conda to PATH"
+
+  - bash: |
+      conda create -q -n p4env python=$PYTHON_VER psi4 --only-deps -c psi4/label/dev
+      source activate p4env
+      which python
+      conda install \
+        dftd3 \
+        gcp \
+        resp \
+        pycppe \
+        pylibefp \
+        snsmp2 \
+        mp2d \
+        pytest-xdist \
+        -c psi4/label/dev
+      conda install \
+        qcelemental \
```

Comment:
I think this needs `blas=*=mkl` and `mkl-include` too.

Reply:
BLAS should come from NumPy, but doesn't hurt to be explicit.

Reply:
Travis was using BLAS/LAPACK from `liblapack-dev`. You might need to export `MATH_ROOT` too.